log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala

```diff
@@ -94,7 +94,11 @@ object LogCollector extends LazyLogging {
 
     httpServerFuture.onComplete {
       case Success(v) => ()
-      case Failure(e) => throw new RuntimeException(e)
+      case Failure(ex) => {
+        ex.printStackTrace()
+        logger.error("Exception caught: " + ex)
```

Comment:
You need to use the right error method here. Dig around in the logging code and look at the different ones. http://www.slf4j.org/api/org/slf4j/Logger.html

You want this one: http://www.slf4j.org/api/org/slf4j/Logger.html#error(java.lang.String,java.lang.Throwable) and you can remove the stacktrace print, it's done automatically in that method

Reply:
I put in the docs for slf4j, we are using scalalogging, but the methods are basically the same.

Reply:
Thanks Tony

Reply:
Thanks Tony. I have changed the method.